app/assets/javascripts/dashboards.coffee

```diff
@@ -1,3 +1,22 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+# File dynamically loads in all patient data on click.
+
+jQuery ->
+  $(document).on "click", ".call_button", ->
+
+    $('.new-call').attr('id', $('.call_button').data('primaryphone'))
+    el = $('.modal-title')
+
+    el.html("")
```

Comment:
I'm sure whatever we do here will involve some coffeescript like this -- I'd love to wrap whatever we do in a named function so it's a little easier to read on the fly, if that's cool with you.